menu_item.cpp

```diff
@@ -217,9 +235,9 @@ MenuItemRoot::MenuItemRoot(const std::vector<std::shared_ptr<TestSuite>> &suites
     child->parent = this;
     submenu.push_back(child);
   }
-
   start_time = std::chrono::high_resolution_clock::now();
 }
+void MenuItemRoot::OnX() { MenuItem::OnX(); }
```

Comment:
Newline before the method, but there's no need to override this if it's just going to call the default. That said, I think you do want it to disable the autorun timer here as well.